account/my_auth.py

```diff
@@ -0,0 +1,33 @@
+from .models      import Account
+from my_settings import SECRET
+
+from django.http  import HttpResponse
+from django.http  import JsonResponse
+
+import jwt
+
+def Authorization(func):
+    def wrapper(self, request, *args, **kwargs):
+        token = request.headers.get('Athorization',None)
```

Comment:
```suggestion
        token = request.headers.get('Authorization',None)
```
테스트 되었는지 확인 부탁 드립니다.